web/utils/analytics-logger.ts

```diff
@@ -0,0 +1,39 @@
+import { AnalyticsLogger, EventEnvironment } from 'bog-analytics';
+
+const developmentLogger = new AnalyticsLogger({ environment: EventEnvironment.DEVELOPMENT });
+const stagingLogger = new AnalyticsLogger({ environment: EventEnvironment.STAGING });
+const productionLogger = new AnalyticsLogger({ environment: EventEnvironment.PRODUCTION });
+
+const apiKey = process.env.NEXT_PUBLIC_BOG_ANALYTICS_CLIENT_API_KEY as string;
+developmentLogger.authenticate(apiKey);
+stagingLogger.authenticate(apiKey);
+productionLogger.authenticate(apiKey);
+
+export function getAnalyticsLogger() {
+    return window.location.href.includes("https://fosters.angelsrescue.org") ? productionLogger
+        : window.location.href.includes("angels-among-us.netlify.app") ? stagingLogger
```

Comment:
Check for `"staging"` instead - `"angels-among-us.netlify.app"` host is used by PR deployment previews as well, which is a development environment

Reply:
This project also has an env variable on Netlify builds `NEXT_PUBLIC_CONTEXT` that provides the environment at runtime